Gruntfile.js

```diff
@@ -10,7 +11,14 @@ module.exports = function(grunt) {
 
   grunt.initConfig({
     pkg: pkg,
-
+    uglify: {
+      my_target: {
+        files: {
+          'dist/sripts/vendor.min.js': ['src/input1.js', 'src/input2.js'],
+          'dist/sripts/app.min.js': ['src/input1.js', 'src/input2.js']
```

Comment:
typo ("sripts")?

I'm still playing with a Babel/Webpack branch and one of the things I ran into is that uglify doesn't play well with ES6. Wanna leave it out for now?